system/testing/BaseTestCase.cfc

```diff
@@ -333,13 +335,33 @@ component extends="testbox.system.compat.framework.TestCase" accessors="true"{
 		var renderedContent = "";
 		var iData			= {};
 
-		// Remove routing, not needed in integration mode.
-		try{
-			getInterceptor( 'SES' ).setEnabled( false );
-		}catch( "InterceptorService.InterceptorNotFound" e ){
-			// ignore it, not used.
-		}
-		
+        if ( arguments.event == "" && arguments.url == "" ){
+            throw( "Must provide either an event or a url to the execute() method." );
+        }
+
+        try{
+            // if we were passed a url, parse it and prepare the SES interceptor for routing.
+            if ( arguments.url != "" ){
+                // separate the url into the url and the query string
+                var urlParts = explodeURL( arguments.url );
+
+                // add the query string parameters to the request context
+                getRequestContext().collectionAppend( urlParts.queryStringCollection );
+
+                // set the integration testing paths so SES routes will be recognized
+                getInterceptor( 'SES' ).setIntegrationTestingPaths(
```

Comment:
Instead of doing this, why don't we leverage MockBox to mock the `getCleanedPaths()` to return the `urlParts.url`?

```
prepareMock( getInterceptor( "SES" ) ).$( "getCleanedPaths", urlParts.url );
```